src/components/TimeseriesDataExport/TimeseriesDataExport.tsx

```diff
@@ -11,15 +11,16 @@ import {
 } from 'antd';
 import { RangePickerValue } from 'antd/lib/date-picker/interface';
 import { FormComponentProps } from 'antd/lib/form';
+import isFunction from 'lodash/isFunction';
```

Comment:
Replace with
```js
import { isFunction } from 'lodash'
```

Reply:
Done; note that this might break tree shaking of lodash in some cases (not sure how you have it configured).

Reply:
we've configured rollup for es6 modules tree shaking